diff --git a/client.py b/client.py
index efcadb6..f2ad499 100644
--- a/client.py
+++ b/client.py
@@ -60,4 +60,4 @@ if __name__ == "__main__":
             prices[stock]=price
             print("Quoted %s at (bid:%s, ask:%s, price:%s)" % (stock, bid_price, ask_price, price))
 
-        print("Ratio %s" % getRatio(prices['ABC'], prices['DEF']))
+        print("Ratio %s" % getRatio(prices['ABC'], prices['DEF']))
\ No newline at end of file
